From: https://gitweb.gentoo.org/repo/gentoo.git/tree/net-fs/nfs4-acl-tools/files/nfs4-acl-tools-0.3.5-jobserver-unavailable.patch

Fix

*      gmake[1]: warning: jobserver unavailable: using -j1.  Add '+' to parent make rule.

--- a/include/buildmacros
+++ b/include/buildmacros
@@ -152,7 +152,7 @@ INSTALL_LINGUAS = \
 endif
 
 SUBDIRS_MAKERULE = \
-	@for d in $(SUBDIRS) ""; do \
+	@+for d in $(SUBDIRS) ""; do \
 		if test -d "$$d" -a ! -z "$$d"; then \
 			$(ECHO) === $$d ===; \
 			$(MAKEF) -C $$d $@ || exit $$?; \
@@ -170,7 +170,7 @@ DIST_MAKERULE = \
 	$(MAKEF) -C build dist
 
 SOURCE_MAKERULE = \
-	@test -z "$$DIR" && DIR="."; \
+	@+test -z "$$DIR" && DIR="."; \
 	for f in $(SRCFILES) ""; do \
 	    if test ! -z "$$f"; then $(ECHO) $$DIR/$$f; fi;\
 	done; \
